src/app/dashboard/dashboard.routing.ts

```diff
@@ -3,10 +3,17 @@ import { RouterModule } from '@angular/router';
 import { DashboardComponent } from './dashboard.component';
 import { DashboardResolveService } from './dashboard-resolve.service';
 import { DashboardEditorComponent } from './dashboard-editor/dashboard-editor.component';
+import { DashboardSensorDetailsComponent } from './dashboard-sensor-details/dashboard-sensor-details.component';
+import { DashboardResolveSensorDetailsService } from './dashboard-sensor-details/dashboard-resolve-sensor-details.service';
 
 export const routes = [
     { path: '', component: DashboardComponent, resolve: {widgets: DashboardResolveService}},
-    { path: 'editor', component: DashboardEditorComponent }
+    { path: 'editor', component: DashboardEditorComponent },
+    {
+        path: 'sensor_details/:id/:period',
```

Comment:
don't we use CONST for routes, don't we, and what an underscore sign in route name?